### PR TITLE
Add support for the muliple widgets

### PR DESCRIPTION
**Pull request by rwx788**
Now we have support of all basic widgets.
I've also added optional parameter for the tables, in case unique value is not in the first column, like it's currently.

Required changes:
* [libyui](https://github.com/libyui/libyui/pull/147)
* [libyui-qt](https://github.com/libyui/libyui-qt/pull/117)
* [libyui-qt-rest-api](https://github.com/libyui/libyui-qt-rest-api/pull/2)
* [libyui-ncurses](https://github.com/libyui/libyui-ncurses/pull/82)
* [libyui-ncurses-rest-api](https://github.com/libyui/libyui-ncurses-rest-api/pull/2)

### CODE REVIEW

**Comment by rwx788** [comment]
@lslezak I messed it up and created PR to my original repo and not fork of `libyui-rest-api`

File: README.md [hunk 1]
```diff
@@ -131,25 +127,34 @@ That means anybody on the way could read the user name and the password
...
+        <li><b>action</b> - action to do</li>
+        <li><b>value</b> (optional) - new value or a parameter of the action</li>
+        <li><b>column</b> (optional) - column id when selecting item in the table</li>
+    </ul>
+</p> Supported actions: <ul>
+    <li><b>press</b> - to press the button</li>
+    <li><b>check</b>|<b>uncheck</b>|<b>toggle</b> - check, uncheck or toggle checkbox</li>
+    <li><b>enter_text</b> - set text in the field, requires <b>value</b> parameter</li>
+    <li><b>switch_radio</b> - activate radio button</li>
+    <li><b>select_combo</b> - select value in the combobox, requires <b>value</b> parameter</li>
```
Comment (jknphy) [inline README.md]: missing to update unique `select` here too.

File: package/libyui-rest-api.changes [hunk 2]
```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Fri Nov 29 14:44:10 UTC 2019 - Rodion Iafarov <riafarov@suse.com>
+
+- Add support to operate on many widgets with rest-api
+- Support column index when selecting a row
+- Update documentation
+- Increase SO vesrion to 11
```
Comment (shundhammer) [inline package/libyui-rest-api.changes]: Typo: version

**Comment by shundhammer** [comment]
LGTM

**Pull request merged**